platform/mwm_version.cpp

```diff
@@ -74,4 +74,13 @@ uint32_t ReadVersionTimestamp(ModelReaderPtr const & reader)
   return version.timestamp;
 }
 
+bool IsSingleMwm(int64_t version)
+{
+  // TODO (mpimenov): Refactor with MwmTraits after merge new-search branch.
+  int64_t const kMinSingleMwmVersion = 151218;
+  // version <= kMaxTestPurposesVersion is reserved for test purposes and is considered as single mwm case.
+  int64_t const kMaxTestPurposesVersion = 5;
```

Comment:
Ok. I agree. 
I've removed kMaxTestPurposesVersion at all
I've added several constants for versions of  mwm single mwm and composed mwm
This constants are used in our mwm tests now